code/modules/food/assembled.dm

```diff
@@ -0,0 +1,221 @@
+
+// Spaghetti + meatball = spaghetti with meatball(s)
+/obj/item/chems/food/boiledspagetti/get_combined_food_products()
+	var/static/list/combined_food_products = list(
+		/obj/item/chems/food/meatball = /obj/item/chems/food/meatballspagetti,
+		/obj/item/stack/nanopaste     = /obj/item/chems/food/nanopasta
+	)
+	return combined_food_products
+
+// Spaghetti with meatballs + meatball = spaghetti with more meatball(s)
+/obj/item/chems/food/meatballspagetti/get_combined_food_products()
+	var/static/list/combined_food_products = list(
+		/obj/item/chems/food/meatball = /obj/item/chems/food/spesslaw
+	)
+	return combined_food_products
+
+/obj/item/chems/food/cutlet/raw/get_combined_food_products()
+	var/static/list/combined_food_products = list(
+		/obj/item/chems/food/meatball/raw = /obj/item/chems/food/sausage
+	)
+	return combined_food_products
+
+/obj/item/chems/food/meatball/raw/get_combined_food_products()
+	var/static/list/combined_food_products = list(
+		/obj/item/chems/food/cutlet/raw = /obj/item/chems/food/sausage
+	)
+	return combined_food_products
+
+/obj/item/chems/food/sausage/get_combined_food_products()
+	var/static/list/combined_food_products = list(
+		/obj/item/chems/food/meatball/raw = /obj/item/chems/food/fatsausage,
+		/obj/item/chems/food/cutlet/raw   = /obj/item/chems/food/fatsausage
+	)
+	return combined_food_products
+
+/obj/item/chems/food/sliceable/flatdough/get_combined_food_products()
+	var/static/list/combined_food_products = list(
+		/obj/item/chems/food/cutlet   = /obj/item/chems/food/meatpie/raw,
+		/obj/item/chems/food/tofu     = /obj/item/chems/food/tofupie/raw,
+		/obj/item/chems/food/xenomeat = /obj/item/chems/food/xemeatpie/raw,
+		"apple"                       = /obj/item/chems/food/applepie/raw,
+		"berries"                     = /obj/item/chems/food/berryclafoutis/raw,
+		"plumphelmet"                 = /obj/item/chems/food/plump_pie/raw
+	)
+	return combined_food_products
+
+// This one looks slightly weird but the attackby() proc should mirror it to the potato.
+/obj/item/chems/food/cheesewedge/get_combined_food_products()
+	var/static/list/combined_food_products = list(
+		"potato" = /obj/item/chems/food/loadedbakedpotato/raw
+	)
+	return combined_food_products
+
+/obj/item/chems/food/fries/get_combined_food_products()
+	var/static/list/combined_food_products = list(
+		/obj/item/chems/food/cheesewedge = /obj/item/chems/food/cheesyfries/raw
+	)
+	return combined_food_products
+
+/obj/item/chems/food/meatpie/raw/cooked_food           = FOOD_RAW
+/obj/item/chems/food/tofupie/raw/cooked_food           = FOOD_RAW
+/obj/item/chems/food/xemeatpie/raw/cooked_food         = FOOD_RAW
+/obj/item/chems/food/applepie/raw/cooked_food          = FOOD_RAW
+/obj/item/chems/food/berryclafoutis/raw/cooked_food    = FOOD_RAW
+/obj/item/chems/food/plump_pie/raw/cooked_food         = FOOD_RAW
+/obj/item/chems/food/loadedbakedpotato/raw/cooked_food = FOOD_RAW
+/obj/item/chems/food/cheesyfries/raw/cooked_food       = FOOD_RAW
```

Comment:
is there no way to specify food that just doesnt give a stressor at all, positive *or* negative? cooking it would still make it *better,* it just wouldn't make you queasy